src/Test.Utilities/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -40,20 +39,6 @@ static Test()
             public Test()
             {
                 ReferenceAssemblies = AdditionalMetadataReferences.Default;
-
-                SolutionTransforms.Add((solution, projectId) =>
-                {
-                    if (AnalyzerConfigDocument is not null)
-                    {
-                        solution = solution.AddAnalyzerConfigDocument(
-                            DocumentId.CreateNewId(projectId, debugName: ".editorconfig"),
-                            ".editorconfig",
-                            SourceText.From($"is_global = true" + Environment.NewLine + AnalyzerConfigDocument),
-                            filePath: @"/.editorconfig");
-                    }
-
-                    return solution;
-                });
```

Comment:
AnalyzerConfigDocument doesn't work with named sections despite it looks like it should parse from the compiler?
This is a point of confusion to me.

While I'm deleting `AnalyzerConfigDocument` altogether, the behavior of it not working when its value is like `"[*]\r\nprop=value` but works when it's `"prop=value"` is very very strange to me.

I'm either missing something or this may indicate a bug from roslyn side.

Reply:
Global config files don't use `[*]`. They just have `is_global=true` as the first line followed by the properties themselves.